plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -163,6 +163,16 @@ protected static function rewritePath($path)
         return SettingsPiwik::rewriteMiscUserPathWithInstanceId($path);
     }
 
+    public static function hasUserLogo()
+    {
+        return file_exists(Filesystem::getPathToPiwikRoot() . '/' . CustomLogo::getPathUserLogo());
```

Comment:
I copied the code from the existing [hasSvgLogo](https://github.com/piwik/piwik/blob/5cf6d07/plugins/CoreAdminHome/CustomLogo.php#L71) and [getPathToLogo ](https://github.com/piwik/piwik/blob/5cf6d07/plugins/CoreAdminHome/CustomLogo.php#L114) methods for consistency :). Is there any difference between the static method and the constant? Should I still use the constant?